contracts/stellar-axelar-operators/src/storage.rs

```diff
@@ -8,3 +8,9 @@ pub enum DataKey {
     #[status]
     Operator { account: Address },
 }
+
+#[contracttype]
+#[derive(Clone, Debug)]
+pub enum MigrationData {
+    Operators(Address),
```

Comment:
Operators contract allows registering one or more operators, so the migration needs to take the full list

Reply:
Avoid using the storage type as part of the public MigrationData type. Storage types are meant to be internal